marshmallow_dataclass/__init__.py

```diff
@@ -367,7 +358,9 @@ def _field_by_type(
 ) -> Optional[Type[marshmallow.fields.Field]]:
     return (
         base_schema and base_schema.TYPE_MAPPING.get(typ)
-    ) or marshmallow.Schema.TYPE_MAPPING.get(typ)
+    ) or marshmallow.Schema.TYPE_MAPPING.get(typ) or (
+        any(kw in str(typ) for kw in ["Serializable", "Mapping", "Dict"]) and dict
+    )
```

Comment:
A hack that translates SerializableAttributes, Mappings, and Dicts into dicts. pretty nifty